ddtrace/contrib/openai/patch.py

```diff
@@ -34,11 +34,14 @@
     "openai",
     {
         "logs_enabled": asbool(os.getenv("DD_OPENAI_LOGS_ENABLED", False)),
+        "llmobs_enabled": asbool(os.getenv("DD_OPENAI_LLMOBS_ENABLED", False)),
```

Comment:
Can we document these new configurations

Reply:
LLM Monitoring is in a private preview for now so this isn't meant to be public. Once it goes GA I'll add it to the docs.